src/main/kotlin/net/ccbluex/liquidbounce/features/command/commands/client/CommandVClip.kt

```diff
@@ -0,0 +1,49 @@
+package net.ccbluex.liquidbounce.features.command.commands.client
+
+import net.ccbluex.liquidbounce.features.command.Command
+import net.ccbluex.liquidbounce.features.command.CommandException
+import net.ccbluex.liquidbounce.features.command.builder.CommandBuilder
+import net.ccbluex.liquidbounce.features.command.builder.blockParameter
+import net.ccbluex.liquidbounce.utils.client.chat
+import net.ccbluex.liquidbounce.utils.client.regular
+import net.ccbluex.liquidbounce.utils.client.variable
+import net.minecraft.client.MinecraftClient
+
+object CommandVClip {
+
+    fun createCommand(): Command {
+        return CommandBuilder
+            .begin("vclip")
+            .parameter(
+                blockParameter("distance")
```

Comment:
I wonder what the point of a block parameter is. Imagine you wrote .vclip stone, what would you expect the command to do? =) Personally, I would prefer to use ParameterBuilder in this case like the other commands do.

Reply:
> I wonder what the point of a block parameter is. Imagine you wrote .vclip stone, what would you expect the command to do? =) Personally, I would prefer to use ParameterBuilder in this case like the other commands do.

Don't know what I can change it to? Using ParameterBuilder seems to completely break the code